### ISSUE TITLE
Rule: Disallow Node modules

### ISSUE THREAD

**Issue by nzakas** [eslint/eslint#791]
This rule would disallow specified modules from being used via `require`. The modules to disallow must be configurable, such as:

``` js
no-restricted-modules: [2, "fs", " child_process "]
```

This would warn on `require("fs")` and `require("child_process ")`
